src/pages/Home.jsx

```diff
@@ -22,14 +22,14 @@ function Home() {
         speakersData={Speakers2024}
         tracks={['AI/ML', 'Mobile', 'Fullstack', 'Miscellaneous']}
       />
+      <SponsorsSection />
+      <OrganizersSection />
+      <FacilitatorsSection />
```

Comment:
I don't recall the ask to shift this layout though I see the shift in the ticket - I'll let @JustinBeaudry make that call